tests/templates/test_layer.py

```diff
@@ -94,10 +95,10 @@ def MultiCircuit(parameters1, parameters2, var1, wires, var2):
     [
         qml.RY(0.5, wires=0),
         qml.RY(0.4, wires=1),
-        qml.templates.BasicEntanglerLayers([[0.5, 0.4]], wires=[0, 1]),
+        qml.templates.BasicEntanglerLayers(np.array([[0.5, 0.4]]), wires=[0, 1]),
         qml.RY(0.5, wires=0),
         qml.RY(0.4, wires=1),
-        qml.templates.BasicEntanglerLayers([[0.5, 0.4]], wires=[0, 1]),
+        qml.templates.BasicEntanglerLayers(np.array([[0.5, 0.4]]), wires=[0, 1]),
```

Comment:
Why do we need the changes in this test file?

Reply:
was going to ask about this. basically since I am using ellipses notation ... on the weight array/tensor I need the weight matrix to be tensor-like (it says in the doc string that they should be anyways) but in these tests, they are passing in nested lists.